homeassistant/components/drop_connect/config_flow.py

```diff
@@ -0,0 +1,103 @@
+"""Config flow for drop_connect integration."""
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING, Any
+
+from dropmqttapi.discovery import DropDiscovery
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.service_info.mqtt import MqttServiceInfo
+
+from .const import (
+    CONF_COMMAND_TOPIC,
+    CONF_DATA_TOPIC,
+    CONF_DEVICE_DESC,
+    CONF_DEVICE_ID,
+    CONF_DEVICE_NAME,
+    CONF_DEVICE_OWNER_ID,
+    CONF_DEVICE_TYPE,
+    CONF_HUB_ID,
+    DISCOVERY_TOPIC,
+    DOMAIN,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+KEY_DEVICE_TYPE = "devType"
+KEY_DEVICE_DESCRIPTION = "devDesc"
+KEY_DEVICE_NAME = "name"
+
+
+class FlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle DROP config flow."""
+
+    VERSION = 1
+
+    _topic = DISCOVERY_TOPIC
```

Comment:
`_topic` isn't used.

Reply:
Removed in 3766113934397bf3d508ec3ddb37f27c6f2d5868.